<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XTC</class>
 <widget class="QMainWindow" name="XTC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>910</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>XTC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_introduction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>introduction</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_introduction">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(116, 115, 117);
font: 75 20pt &quot;Arial&quot;;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>5</number>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>.......&lt;&gt;&lt;&lt;P</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_introduction">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>20</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>XTC introduction</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_rawdata">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>load_data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>reading data files</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextBrowser" name="textBrowser_xtci">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>show some wrong message</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_matrix">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>show_data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget_matrix">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_next">
            <property name="text">
             <string>next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_plot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>plots</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" rowspan="5">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>20</y>
              <width>451</width>
              <height>691</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayoutCheckbox">
             <item>
              <widget class="QGroupBox" name="groupBoxCore">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Cores</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_params">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>params</string>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>groupBox_params</zorder>
            <zorder>groupBoxCore</zorder>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2">
           <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>1061</width>
              <height>611</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayoutPlots">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
           <widget class="QChartView" name="chart" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>650</y>
              <width>1061</width>
              <height>121</height>
             </rect>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>620</y>
              <width>1061</width>
              <height>48</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="paramPrevo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(255, 0, 128);
background-color: rgba(255, 255, 255, 0);
border-color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>&lt;|-</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="paramNext">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(255, 0, 128);
background-color: rgba(255, 255, 255, 0);
border-color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>-|&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <zorder>layoutWidget</zorder>
           <zorder>chart</zorder>
           <zorder>verticalLayoutWidget</zorder>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_button">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pBhideOp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(255, 0, 128);
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBshowOp">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>5</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTipDuration">
             <number>0</number>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(255, 0, 128);
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_back_edit">
            <property name="styleSheet">
             <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(255, 0, 128);
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" rowspan="4">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_edit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(51, 51, 51);</string>
       </property>
       <attribute name="title">
        <string>edit</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pBshowOp_E">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(255, 0, 128);
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPlots_E">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <item>
               <widget class="QPushButton" name="paramPrevo_E">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(255, 0, 128);
background-color: rgba(255, 255, 255, 0);
border-color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>&lt;|</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="paramNext_E">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(255, 0, 128);
background-color: rgba(255, 255, 255, 0);
border-color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>|&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1203</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_f">
    <property name="title">
     <string>File &amp;f</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen_project_file"/>
    <addaction name="actionCreat_new_project"/>
    <addaction name="actionSave_correlation_models"/>
   </widget>
   <widget class="QMenu" name="menuView_v">
    <property name="title">
     <string>view &amp;v</string>
    </property>
    <addaction name="actionCore_Mode"/>
    <addaction name="actionParam_mode"/>
    <addaction name="actionMater_record"/>
    <addaction name="actionEdit_Mode"/>
   </widget>
   <widget class="QMenu" name="menuTool_t">
    <property name="title">
     <string>tool &amp;t</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>quit</string>
    </property>
    <addaction name="actionQuit_and_save_data"/>
    <addaction name="actionAbort"/>
   </widget>
   <addaction name="menuFile_f"/>
   <addaction name="menuView_v"/>
   <addaction name="menuTool_t"/>
   <addaction name="menuQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_project_file">
   <property name="text">
    <string>open project file</string>
   </property>
  </action>
  <action name="actionCreat_new_project">
   <property name="text">
    <string>creat new project</string>
   </property>
  </action>
  <action name="actionSave_correlation_models">
   <property name="text">
    <string>save correlation models</string>
   </property>
  </action>
  <action name="actionParam_mode">
   <property name="text">
    <string>param mode</string>
   </property>
  </action>
  <action name="actionMater_record">
   <property name="text">
    <string>mater record</string>
   </property>
  </action>
  <action name="actionQuit_and_save_data">
   <property name="text">
    <string>quit and save data</string>
   </property>
  </action>
  <action name="actionAbort">
   <property name="text">
    <string>abort</string>
   </property>
  </action>
  <action name="actionEdit_Mode">
   <property name="text">
    <string>Edit Mode</string>
   </property>
  </action>
  <action name="actionCore_Mode">
   <property name="text">
    <string>core Mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
